packages/eslint-plugin-pf-codemods/lib/rules/v5/formgroup-remove-helpertextProps.js

```diff
@@ -0,0 +1,29 @@
+const { getPackageImports } = require("../../helpers");
+
+// https://github.com/patternfly/patternfly-react/pull/8810
+module.exports = {
+  meta: {},
+  create: function (context) {
+    const formGroupImport =  getPackageImports(context, '@patternfly/react-core')
+    .filter(specifier => specifier.imported.name == 'FormGroup');
+
+    return formGroupImport.length === 0 ? {} : {
+      JSXOpeningElement(node) {
+        if (formGroupImport.map(imp => imp.local.name).includes(node.name.name)) {
+          const helperProps = node.attributes.filter(a => {
+            const attr = a.name?.name;
+            return attr === 'helperText' || attr === 'helperTextInvalid' || attr === 'validated' || attr === 'helperTextIcon' || attr === 'helperTextInvalidIcon' || attr === 'isHelperTextBeforeField'
```

Comment:
Nit: could do an array of all the strings here and check if it `.includes(a.name.name)` instead